Stardrop/Models/Data/ModDownloadEvents.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Threading;
+
+namespace Stardrop.Models.Data
+{
+    internal record ModDownloadStartedEventArgs(Uri Uri, string Name, long? Size, CancellationTokenSource DownloadCancellationSource);    
```

Comment:
One thing I noticed while working--some things seem to use `internal` visibility, while others are `public`. It seemed as though `internal` was a hint that "this is limited to some particular subsystem", and I tried to follow that convention here.